lesson13/script.js

```diff
@@ -55,27 +55,35 @@ async function init(){
     }
 }
 
+(function(){
+    const modal = document.getElementById('js-modal');
+    const openButton = document.getElementById('js-open-btn');
+    const closeButton = document.getElementById('js-close-btn');
+    const requestButton = document.getElementById('js-request-btn');
+
 function openModalWindow(){   
-    document.getElementById('js-open-btn').addEventListener('click',function(){
-        document.getElementById('js-modal').classList.add('is-show');
+    openButton.addEventListener("click",function () {
+        modal.classList.add("is-show");
         this.classList.add('is-deactive');
     },false);
 }
 
 function closeModalWindow(){ 
-    document.getElementById('js-close-btn').addEventListener('click',function(){
-        document.getElementById('js-modal').classList.remove('is-show');
-        document.getElementById('js-open-btn').classList.remove('is-deactive');
+    closeButton.addEventListener("click", function () {
+        modal.classList.remove("is-show");
+        openButton.classList.remove("is-deactive");
     },false);
 }
 
 function requestData(){ 
-    document.getElementById('js-request-btn').addEventListener('click', function(){
-        document.getElementById('js-modal').classList.remove('is-show');
+    requestButton.addEventListener('click', function(){
+        modal.classList.remove('is-show');
         init();
     },false);
 }
 
 openModalWindow();
 closeModalWindow();
 requestData();
+
+}());
```

Comment:
I changed to the method of using the immediate function that  Haru-san taught me.

Reply:
I unified to single quotation.
https://github.com/nakagawamai/js-lesson/pull/15/commits/359b677e3a45d32f11200b57104e81d5ca1d5b1e

Reply:
I delete immediate function.
Because I was using immediate functions to reduce global variables.
https://github.com/nakagawamai/js-lesson/pull/15/commits/45d4a0455eb52dcf344f9302addf01c0d17b86d8